### ISSUE TITLE
Any plans to update this to .Net Core? And use Tag Helpers instead of HTML Helpers?

### ISSUE THREAD

**Issue by egmfrs** [ajsaulsberry/BlipAjax#1]
This would be really useful to me.

**Comment by ajsaulsberry** [comment]
Yes, it's in the works. (But no ETA yet because of other responsibilities.) I'm planning on including additional standards, such as dependency injection, in the updated example. I will update this issue when I have more information to share.
